### PR TITLE
WIP: Reorganized content

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

For issue [#2182](https://github.com/eclipse/codewind/issues/2182) 

@james-wallis, thank you for the clarification and please review.

### CODE REVIEW

**Comment by james-wallis** [comment]
Hi @j-c-berger, I think something like the following will read better as the generic Docker types still needs to follow the "Requirements for importing an existing project not created in Codewind or through Appsody or Odo" section.

I've marked it up so you can copy it over if you agree.

Thanks

```
## Requirements for importing an existing project not created in Codewind or through Appsody or Odo
Local projects are built and run on Docker. To import your project to a local Codewind instance, it must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
*Note: The Dockerfile must contain an `EXPOSE` instruction to point to the port project will run on*.

Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, it must include a Dockerfile and a Helm chart. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and for more information about Helm charts, see [Helm chart documentation](https://helm.sh/docs/topics/charts/).


## What kind of projects can I add?
Codewind is designed to develop cloud-native microservices. Therefore, each project must be self-sufficient, so not reliant on other projects to build. 

To import your own project into Codewind you only need to follow the requirements outlined in the previous section. Make sure to select the **Other (Basic Container)** option as the project type.

Codewind also contains a special build engine for the following languages. To utilise the build engine follow the setup guides below.
* [MicroProfile/Java EE projects](#microprofilejava-ee-projects)
* [Java Spring projects](#java-spring-projects)
* [Node.js projects](#nodejs-projects)
* [Swift projects](#swift-projects)
* [Appsody projects](#appsody-projects) 
```

**Comment by j-c-berger** [comment]
@james-wallis, thanks this is great! I'll take a closer look and get a new commit up later today

**Comment by j-c-berger** [comment]
@james-wallis, please see the new commit, thanks!

**Comment by james-wallis** [comment]
Just checked the latest changes, I don't think the `Generic Docker projects` section is need at all now as the same content is covered in the `Requirements for importing an existing project not created in Codewind or through Appsody or Odo` and `What kind of projects can I add?` sections. Apart from that its all good.

**Comment by j-c-berger** [comment]
@sishida, per James' comment, I deleted ```Generic Docker projects``` section in the new commit above. 

Please review then merge when you have the time, thanks!

File: docs/_documentations/importing-existing-projects.md [hunk 1]
```diff
@@ -15,24 +15,33 @@ Configure a project to run in Codewind.
...
+Local projects are built and run on Docker. To import your project to a local Codewind instance, you must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
+**Note:** The Dockerfile must contain an `EXPOSE` instruction to point to the port that the project runs on.
+
+Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, you must include a Dockerfile and a Helm chart. For more information about Dockerfiles and Helm charts, see: 
 
-Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, it must include a Dockerfile and a Helm chart. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and for more information about Helm charts, see [Helm chart documentation](https://helm.sh/docs/topics/charts/).
+* [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/)
+* [Helm chart documentation](https://helm.sh/docs/topics/charts/).
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md:30]: No period needed at the end of the bullet point.

File: docs/_documentations/importing-existing-projects.md [hunk 2]
```diff
@@ -15,24 +15,33 @@ Configure a project to run in Codewind.
...
 
 ## What kind of projects can I add?
 
-Codewind is designed to develop cloud-native microservices. Therefore, each project must be self-sufficient, so not reliant on other projects to build. The requirements to add projects for each of the supported application types are outlined in the following sections:
+Codewind develops cloud-native microservices. Therefore, each project must be self-sufficient, so not reliant on other projects to build.
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md:34]: I'd delete "so"...it's a conjunction, but there's no complete phrase on the other side of this comma + conjunction pairing.

File: docs/_documentations/importing-existing-projects.md (hunk 1 above)
Comment (sishida) [reply docs/_documentations/importing-existing-projects.md:30]: Or put it on one line: "...and Helm charts, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and [Helm chart documentation](https://helm.sh/docs/topics/charts/)."

I kind of prefer this way.

**Comment by j-c-berger** [comment]
@sishida, thanks for the review! Please merge if it all looks good to you

**Pull request merged**